ArchUnitNET/Fluent/Syntax/Elements/Members/IMembersShould.cs

```diff
@@ -0,0 +1,28 @@
+﻿using ArchUnitNET.Domain;
+
+namespace ArchUnitNET.Fluent.Syntax.Elements.Members
+{
+    public interface
```

Comment:
Why do you need these interfaces?

Reply:
I wanted to use these interfaces to write the documentary in the future.